openmdao_book/features/building_blocks/components/external_code_implicit_comp.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<p>I don't know what this block is for. It isn't in the original.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>This is from the <code>extcode_mach</code>  file. Because it was in a separate file, the functions wouldn't get "stored" in the notebook so it would error out. I added the code from the original doc but had to put it in markdown format due to failures if it was a runable cell. To fix this, I added the block in question and hid it as a helper cell</p>